ctl/pre_sort.go

```diff
@@ -23,27 +26,37 @@ type PreSortCommand struct {
 
 	Table string `json:"table"`
 
-	PrimaryKeyFields []string `json:"primary-key-fields"`
+	PrimaryKeyFields    []string `json:"primary-key-fields"`
+	PrimaryKeySeparator string
```

Comment:
does this need a json tag?

Reply:
actually I think all these tags are completely spurious... I put them in out of habit because commandeer will read them... but we don't use commandeer for FB subcommands

Reply:
removed